amida-tech/indaba-client
# Bug/inba 693 jf scale question fix

#### What does this PR do?
There was an issue with scale type questions, where trying to manually type in a number would validate it too rapidly. For example, if the scale goes from 5 to 50, and the user enters the first 4 to a value of 49, the `onBlur` would assume the 4 was the full value and reject it.

This change stores the value with an `onChange` sans validation, _then_ validates `onBlur`. 

Also, went ahead and got rid of a `required` argument we don't need anymore. Yay4refactoring.

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-693

#### How should this be manually tested?
Create a survey with a scale type question and assign it. Login as that user and try entering a value with a first digit that would be outside the min max bounds. Check that it only rejects and resets the value to minimum if you navigate AWAY from the field. 

#### Background/Context

#### Screenshots (if appropriate):
